pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -1539,6 +1539,7 @@ default-package-overrides:
   - palette ==0.3.0.2
   - pandoc ==2.7.3
   - pandoc-citeproc ==0.16.2
+  - pandoc-crossref ==0.3.4.2
```

Comment:
This should be moved to the top of this list.  See the comment on this list.

Alternatively, if you wanted to get `pandoc-crossref` in stackage, we wouldn't have to specifically deal with this in nixpkgs.

Reply:
Sorry, I didn't notice. Done.

I don't know the process: should I just open an issue in [commercialhaskell/stackage](https://github.com/commercialhaskell/stackage)?

Reply:
The stackage homepage has a few links with info on adding a package:

https://www.stackage.org/

In particular:

- https://github.com/commercialhaskell/lts-haskell#readme
- https://github.com/commercialhaskell/stackage#frequently-asked-questions
- https://github.com/commercialhaskell/stackage/blob/master/MAINTAINERS.md#adding-a-package